platform/admin/core/admin-core-insecuredefaults/src/main/java/org/codice/ddf/admin/insecure/defaults/service/InsecureDefaultsCollector.java

```diff
@@ -55,8 +63,8 @@
   private static final String SERVER_SIGNATURE_PROPERTIES_FILE =
       new AbsolutePathResolver("etc/ws-security/server/signature.properties").getPath();
 
-  private static final String USERS_PROPERTIES_FILE =
-      new AbsolutePathResolver("etc/users.properties").getPath();
+  private static final Path USERS_PROPERTIES_FILE_PATH =
```

Comment:
If we want a path instead of a string we should just wrap the current `new AbsolutePathResolver(...).getPath()` in a  `Paths.get` call instead of pulling it from the scheduler. Using a Path type here does break with the style of the rest of the paths in this class though.

Reply:
I did that so we don't define the path in different places. In `UserPropertiesDeletionScheduler`, I have
```static final Path USERS_PROPERTIES_FILE_PATH =
      Paths.get(new AbsolutePathResolver("etc/users.properties").getPath());```
I don't mind changing it back to a string and adding another constant that defines the Path.